Retail.lua

```diff
@@ -343,17 +371,18 @@ addon.Spells = {
     [81782] = { type = BUFF_DEFENSIVE }, -- Power Word: Barrier
     [87204] = { type = CROWD_CONTROL }, -- Sin and Punishment
     [194249] = { type = BUFF_OFFENSIVE }, -- Voidform
-    [196762] = { type = BUFF_DEFENSIVE }, -- Inner Focus
     [232707] = { type = BUFF_DEFENSIVE }, -- Ray of Hope
     [197862] = { type = BUFF_DEFENSIVE }, -- Archangel
-    [197871] = { type = BUFF_OFFENSIVE }, -- Dark Archangel
+    [197871] = { type = BUFF_OFFENSIVE }, -- Dark Archangel (on the priest)
+        --[197874] = { type = BUFF_OFFENSIVE, parent = 197871 }, -- Dark Archangel (on others), might make it harder to see other offensives being used
```

Comment:
I think that people using their own offensive CDs should draw on top of a Dark, and not seeing them do that _because_ of a Dark being drawn over it might be catastrophic - this is why I ended up commenting it out, because there's a catastrophic scenario with it left on (although I'm not entirely sure what the addon will pick if there's several things with the same priority).

Having it on a priority number that is just below all the other offensive CDs would be ideal. The only problem is that a user might have custom priorities set up, so we should do something like "check the config, grab what the OFFENSIVE_BUFFS is set to, and make yourself -1 from that". But then people might change those as well and not realize that DA isn't changing relatively to their new setting, and there's nothing that tells them that we have one spell that comes with "custom priority" ticked on out of the box... I'm starting to wonder if just putting it into "BUFFS_OTHER" and leaving a comment in the code explaining WHY its there is the way to go, since most people will actually have BUFFS_OTHER on a lower priority than BUFFS_OFFENSIVE.